db/migrate/20220824185245_add_unconfirmed_to_devise.rb

```diff
@@ -0,0 +1,16 @@
+class AddUnconfirmedToDevise < ActiveRecord::Migration[6.1]
+  def change
+    change_table(:users) do |t|
+      t.string   :confirmation_token
+      t.datetime :confirmed_at
+      t.datetime :confirmation_sent_at
+      t.string   :unconfirmed_email # Only if using reconfirmable
+  
+        ## Lockable
+      #t.integer  :failed_attempts, default: 0, null: false # Only if lock strategy is :failed_attempts
+      #t.string   :unlock_token # Only if unlock strategy is :email or :both
+      #t.datetime :locked_at
+      # end
+    end
+  end
+end
```

Comment:
- [ ] Kindly note that all the logic of confirmable was done in the `add_devise_to_user` migration, allow me to say that this migration is useless for now. here is the confirmable already done in the migration `add_devise_to_user` logic :point_down: 

```rb
# frozen_string_literal: true

class AddDeviseToUsers < ActiveRecord::Migration[6.1]
  def self.up
    change_table :users do |t|
      ## Database authenticatable
      t.string :email,              null: false, default: ""
      t.string :encrypted_password, null: false, default: ""

      ## Recoverable
      t.string   :reset_password_token
      t.datetime :reset_password_sent_at

      ## Rememberable
      t.datetime :remember_created_at

      ## Trackable
      # t.integer  :sign_in_count, default: 0, null: false
      # t.datetime :current_sign_in_at
      # t.datetime :last_sign_in_at
      # t.string   :current_sign_in_ip
      # t.string   :last_sign_in_ip

      ## Confirmable
      t.string   :confirmation_token
      t.datetime :confirmed_at
      t.datetime :confirmation_sent_at
      t.string   :unconfirmed_email # Only if using reconfirmable

      ## Lockable
      # t.integer  :failed_attempts, default: 0, null: false # Only if lock strategy is :failed_attempts
      # t.string   :unlock_token # Only if unlock strategy is :email or :both
      # t.datetime :locked_at


      # Uncomment below if timestamps were not included in your original model.
      # t.timestamps null: false
    end

    add_index :users, :email,                unique: true
    add_index :users, :reset_password_token, unique: true
    # add_index :users, :confirmation_token,   unique: true
    # add_index :users, :unlock_token,         unique: true
  end

  def self.down
    # By default, we don't want to make any assumption about how to roll back a migration when your
    # model already existed. Please edit below which fields you would like to remove in this migration.
    raise ActiveRecord::IrreversibleMigration
  end
end
```

Reply:
Thank you @andersonlebon, I see what you mean. I have fixed that.
That's two reviews gone out of three 😢 🥺